packages/page-pattern-modal/src/styles/page-pattern-modal.scss

```diff
@@ -53,15 +65,20 @@ $sidebar-width-desktop: 324px;
 	.components-modal__content {
 		overflow-y: scroll;
 		-webkit-overflow-scrolling: touch;
-		// prevents a 1px strip of content being visible after scrolling
-		margin-top: -1px;
+		margin-top: 60px;
 		padding: 0 16px 16px;
+
+		&::before {
+			margin-bottom: 0;
+		}
 	}
 
 	.components-modal__header {
 		border-bottom: none;
 		margin-right: -16px;
+		background-color: #fff;
```

Comment:
Was the header inheriting a different color in some circumstances?

Reply:
ah, so its not transparent when the z-index puts it above scrolling content? 👍

Reply:
Nope, just blindly adding stuff that was [suggested here](https://github.com/Automattic/wp-calypso/issues/53754#issuecomment-862476885). I thought I'd seen it bleeding through, too, but it's good to remove.